### PR TITLE
[IMP] purchase: replace purchase uom screenshots

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to the `uom` doc in Odoo's Purchase documentation, specific to 17.0.

Here's what I did:
- Replaced all screenshots for 17.0 UI, removed some screenshots no longer relevant

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

[Project Task Link](https://www.odoo.com/web#cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form&id=3926218)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @jero-odoo, real quick PR ready for you to review - just replaced the screenshots on this one. Thank you!

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/purchase/products/uom.rst:77]: Early line break (move category to line 77)

Comment (jero-odoo) [inline content/applications/inventory_and_mrp/purchase/products/uom.rst:77]:
```suggestion
measure inside the category. Name the unit, and select the units of measure
```

**Review by jero-odoo — APPROVED** [review]
Hey @brse-odoo just a quick note, other than that this is ready for the next round. Thanks!

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/purchase/products/uom.rst:77]: Suggestion implemented ✅

Comment (brse-odoo) [reply content/applications/inventory_and_mrp/purchase/products/uom.rst:77]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
@odoo/us-doc-review Real quick PR ready for review - just replaced the screenshots on this one. Thank you!

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/products/uom.rst:6]:
```suggestion
when it is sold This can cause confusion between sales and purchase representatives. It is
```
please ensure the rest of the doc removes any instance of "you" or "your" -- Thanks

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/products/uom.rst:74]:
```suggestion
The next step is to create the two units of measures. To do so, click into the :guilabel:`Unit of Measure
Category` field and enter a name for the category. Then, under the :guilabel:`Units of Measure` tab, click :guilabel:`Add a line`.
```

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/purchase/products/uom.rst:80]: remove the use of "you" and correct the incorrect usage of italics here, please

**Review by ksc-odoo — APPROVED** [review]
The images look good to me, which I can see is the main purpose of this PR. Approving now solely for those.

However...

I fully understand that you are only focusing on the screenshots here, but there is a decent amount of formatting issues located throughout this doc that could use a much-needed update (e.g. "you"/"your mentions", incorrect menuselection formatting, etc.)

So, if you don't get to those this time around, I strongly suggest creating a new PR in the near future to address those issues. Thanks!

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/purchase/products/uom.rst:6]: Changing this specific instance in this suggestion, but I am not making any other updates to the copy outside of it, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/purchase/products/uom.rst:74]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/purchase/products/uom.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/purchase/products/uom.rst:80]: I am not making any other updates to the copy outside of the one implemented suggestion, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.

**Comment by brse-odoo** [comment]
> Hey, @brse-odoo
> 
> The images look good to me, which I can see is the main purpose of this PR. Approving now solely for those.
> 
> However...
> 
> I fully understand that you are only focusing on the screenshots here, but there is a decent amount of formatting issues located throughout this doc that could use a much-needed update (e.g. "you"/"your mentions", incorrect menuselection formatting, etc.)
> 
> So, if you don't get to those this time around, I strongly suggest creating a new PR in the near future to address those issues. Thanks!

Hi @ksc-odoo, thanks for the review.

Since this was a PR only to update the screenshots, I am not making any other updates to the copy outside of the one implemented suggestion, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged. Thank you

**Comment by brse-odoo** [comment]
Hi @samueljlieber Real quick PR ready for review - just replaced the screenshots on this one. Thank you!

**Review by samueljlieber — APPROVED** [review]
Thanks for the improvements @brse-odoo! Looks good to me 👍
..
@robodoo r+

**Pull request closed without merging**
